HelpSource/Classes/MIDIOut.schelp

```diff
@@ -141,9 +141,15 @@ m.connect(1);  // connect to MIDIClient.destinations[1]
 
 WARNING:: The teletype::port:: argument to link::Classes/MIDIOut#*new:: has an entirely different meaning in Linux, compared to macOS and Windows. If user code calls this method emphasis::and:: cross-platform compatibility is needed, it is the user's responsibility to handle Linux separately. User code can check the platform using code::thisProcess.platform.name:: (which returns one of code::\osx::, code::\linux:: or code::\windows::). Or, for compatibility, use link::#*newByName:: instead.::
 
-link::Classes/MIDIOut#*new:: optionally takes a second argument, for the uid of the MIDI destination device. This establishes a direct connection to the target, which is not visible in Qjackctl. The teletype::port:: argument is irrelevant in this case; the MIDIOut instance is always one-to-one with the target.
+link::Classes/MIDIOut#*new:: optionally takes a second argument, for the uid of the MIDI destination device. If the uid is non-zero, then: 1/ all connections in the ALSA MIDI patchbay are ignored, and 2/ individual messages will be sent directly to the specified device.
 
-link::Classes/MIDIOut#*newByName:: should create one-to-one MIDIOut connections on all platforms.
+link::Classes/MIDIOut#*newByName:: locates a device by name and populates the uid in the new MIDIOut object. Thus uid should be non-zero for all MIDIOut objects created using this method.
```

Comment:
>  "Thus uid should be non-zero for all MIDIOut objects (...)"

On first reading I had an impression that "should" referred to requiring the user to provide these. What do you think about changing it to something along the lines of "Thus uid will be set to non-zero (...)"? Or maybe even refer to it as "internally used uid"? (not sure about that last one)

Reply:
Fixed.